### PR TITLE
Add Coming Soon to Site Features

### PR DESCRIPTION
**Pull request by officiallygod**
ss

### CODE REVIEW

**Comment by officiallygod** [comment]
Requires: https://github.com/newfold-labs/wp-module-onboarding-data/pull/5

File: src/OnboardingSPA/utils/api/comingSoon.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { resolve } from './resolve';
+import { wpSiteUrl } from '../../../constants';
+
+import apiFetch from '@wordpress/api-fetch';
+
+export async function setComingSoon( comingSoon ) {
+	const data = { comingSoon };
+	return await resolve(
+		apiFetch( {
+			url: wpSiteUrl + '/index.php?rest_route=/bluehost/v1/settings',
```
Comment (arunshenoy99) [inline src/OnboardingSPA/utils/api/comingSoon.js:10]: https://github.com/newfold-labs/wp-module-onboarding/blob/cdbd7f7a229e96935bd7d7a04bfa36b07ebda77f/src/OnboardingSPA/utils/api/ecommerce.js#L10

Comment (officiallygod) [reply src/OnboardingSPA/utils/api/comingSoon.js:10]: Thanks, let me take a look!

**Review by abhijitb — APPROVED** [review]
LGTM!

**Pull request merged**